src/core/components/InputAdornmentMessage.module.css

```diff
@@ -4,13 +4,22 @@
   gap: var(--spacing-sm);
   padding: var(--spacing-sm);
   width: 98%;
-  background-color: var(--colors-accentTwoMuted);
   font-size: 1.1rem;
   color: var(--colors-textPrimary);
   margin-bottom: -1rem;
   border-radius: var(--borderRadius-sm);
 }
 
+.cautionBanner {
+  composes: banner;
+  background-color: var(--colors-accentTwoMuted);
+}
+
+.warningBanner {
+  composes: banner;
+  background-color: var(--colors-accentMuted);
+}
+
```

Comment:
Diff here is a little weird

- the existing `banner` class is no longer used directly in the component
- it acts as a base class for the two new ones: caution & warning
- the only difference between those two is the background color, so this keeps the CSS DRY'er